.env.example

```
@@ -1,15 +1,23 @@
 
 # ChromaDB Configuration
 CHROMADB_URL=http://localhost:8000
+CHROMA_PORT=8000
+
+# AI Service API Keys
+ANTHROPIC_API_KEY=key
+OPENAI_API_KEY=your_openai_api_key_here
+
+# Server Configuration
+PORT=3000
+NODE_ENV=development
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the PORT key

(UnorderedKey)